### PR TITLE
Translate Carousel's documentation to English

### PR DESCRIPTION
**Pull request by superRaytin**
#1471

### CODE REVIEW

File: components/carousel/demo/fade.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 2
-title: 渐显
+title:
+  zh-CN: 渐显
+  en-US: Fade
```
Comment (benjycui) [inline components/carousel/demo/fade.md]: Fade In ?

**Comment by benjycui** [comment]
@manV please review this translation, thanks! 😀

File: components/carousel/demo/fade.md (hunk 1 above)
Comment (superRaytin) [inline components/carousel/demo/fade.md]: Yes, you're right!

File: components/carousel/demo/vertical.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: Vertical
 ---
 
+## zh-CN
+
 垂直显示。
 
+## en-US
+
+Use a vertical display.
```
Comment (manV) [inline components/carousel/demo/vertical.md]: Vertical pagination

File: components/carousel/index.en-US.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+category: Components
+type: Presentation
+title: Carousel
+---
+
+A carousel component. Scales with its container.
+
+## When to use
+
+- There is a group of content on the same level.
```
Comment (manV) [inline components/carousel/index.en-US.md]: **When** there is a group of content on the same level.

File: components/carousel/index.en-US.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- There is insufficient content space, it can be used to save space in the form of a resolving door.
```
Comment (manV) [inline components/carousel/index.en-US.md]: **When** there is insufficient content space, it can be used to save space in the form of a resolving door.

**Comment by benjycui** [comment]
@superRaytin 根据 review 更新下？

**Comment by benjycui** [comment]
Cool!

**Pull request merged**